.travis.yml

```diff
@@ -18,12 +18,16 @@ matrix:
   include:
     - name: Test
       script:
+        - cargo install moleculec
+        - make c/protocol_reader.h
         - make all-via-docker
-        - cargo test --all
+        - cargo test --tests
```

Comment:
`cargo test --all` crashes randomly when linking doc-tests in my laptop, although this crate doesn't have any doc-tests.
(I spent a lot of time on this.) I think there is a bug in rust toolchain or any dependencies, so I updated this.